### PR TITLE
Capston  project

### PR DESCRIPTION
**Pull request by ademibowale**
## HTML/CSS & JavaScript capstone project - Conference page

- [ ] Here is the loom video link  where i explain about this capstone project 
   https://www.loom.com/share/201a1c930c084c29a9f3cbf1198e59bb
- [ ] My code follows the style guidelines of this project
- [ ] Implement  javscript functionality 
- [ ] I have performed a self-review of my code
- [ ] I personalize the content of the page. I Choose a topic that is different than the one in the original design.
- [ ] I follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:

     - Colors.
     - Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
     - Layout: composition and space between elements.
     - The pages look almost identical to the original design. Small adjustments like text or image changes are acceptable.

     - I use a CSS framework (for example Bootstrap) for styling.
- [ ] I build only these 2 pages:

     - The home page.
     - The about page.
- [ ] Each of these pages have versions for 2 different screen sizes:
     - Mobile: up to 768px wide.
     - Desktop: 768px or wider.
     - Interactions

- [ ] The home page have a link in the menu to the about page.
- [ ] The logo in the header links to the home page.
Mobile menu
- [ ] When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- [ ] There are no guidelines for the mobile menu in the docs, but i implemented it so it is consistent with the design (colors, typography, spacings, etc.).
- [ ] The mobile menu have a close (X) button that closes the menu.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings

## LIVE DEMO:
https://ademibowale.github.io/Capstone_projects/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,79 @@
...
+
+
+## Built With
+
+- HTML
+- CSS 
+- JAVASCRIPT
+
+## Live Demo
+
```
Comment (Kossi-stack) [inline README.md]: - [x] Kindly remove the empty spaces for clean code and also provide the live demo link of the your project.

File: README.md [hunk 2]
```diff
@@ -1 +1,79 @@
...
+Run npm install.
+Test:
+
+run the project in the browser.
+Use google tools to inspect the web into mobile size.
+
+## Author
+
+👤 **ADEBOWALE ADEGBOYE**
+
```
Comment (Kossi-stack) [inline README.md:53]: - [ ] Kindly fix the social handles
Capturemedia

File: README.md [hunk 3]
```diff
@@ -1 +1,79 @@
...
+# Capstone_projects
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Microverse Capstone Project
+
+ This project is the Final capstone project of my first module at microverse. It is aim at building responsive UI of an online website for  Music Entertainment Industry , so in this project I built the home and the about page of the conference page (and readable) on all screens.
+The template I used to create this template was originally designed by [Cindy Shin in Behance.](https://www.behance.net/adagio07)
+
+https://github.com/ademibowale/Capstone_projects
+https://www.loom.com/share/201a1c930c084c29a9f3cbf1198e59bb
```
Comment (Kossi-stack) [inline README.md]: - [x] Kindly remove the loom video here and put it in your PR message as a link.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+                              </ul>
+                              </nav>
+                              </header>
+          <section id="home">
+          <div class="main-sectionA  desktopsecA">
+          <h2 class="head"> "Hello! Sharing World" <br> <br>
+          HYCROWN ENTERTAINMENT CREATIVE COMMON GLOBAL
+          SUMMIT 2022</h2>
+          <p class="para-text paratextA ">
+          A fun festival where people from more than 80
```
Comment (Kossi-stack) [inline index.html:50]: - [ ] Kindly remove the white space here
Capturemobblank

File: index.html [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+                                                            people.
+                                                            It carries a message, and we, the musicians, are the
+                                                            messengers." </p>
+                                        </li>
+                              </ul>
+                              <a href="#" class="btn flex" role="button">OUR PROGRAMMES</a>
+                    </section>
+                    <section id="cart">
+                              <div id="speakers-about" class="flex-column">
+                                        <h3>Featured Speakers</h3>
```
Comment (Kossi-stack) [inline index.html]: - [x] Kindly fix the image
Capturefiximage

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @ademibowale,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 4 above)
Comment (ademibowale) [reply index.html:50]: thank you so much i have fix it perhaps the slideshow images not appear but i have fix this  . thank you

File: README.md (hunk 1 above)
Comment (ademibowale) [reply README.md]: please we have no demo link for this project yet until its approve

**Comment by ademibowale** [comment]
## In this Milestone...

## HTML/CSS & JavaScript capstone project - Conference page

- [ ] here is the loom video link  where i explain about this capstone project 
   https://www.loom.com/share/201a1c930c084c29a9f3cbf1198e59bb
 
- [ ] My code follows the style guidelines of this project
- [ ] Implement  javscript functionality 
- [ ] I have performed a self-review of my code
- [ ] I personalize the content of the page. I Choose a topic that is different than the one in the original design.
- [ ] I follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:

     - Colors.
     - Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
     - Layout: composition and space between elements.
     - The pages look almost identical to the original design. Small adjustments like text or image changes are acceptable.

     - I use a CSS framework (for example Bootstrap) for styling.
- [ ] I build only these 2 pages:

     - The home page.
     - The about page.
- [ ] Each of these pages have versions for 2 different screen sizes:
     - Mobile: up to 768px wide.
     - Desktop: 768px or wider.
     - Interactions

- [ ] The home page have a link in the menu to the about page.
- [ ] The logo in the header links to the home page.
Mobile menu
- [ ] When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- [ ] There are no guidelines for the mobile menu in the docs, but i implemented it so it is consistent with the design (colors, typography, spacings, etc.).
- [ ] The mobile menu have a close (X) button that closes the menu.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings

## LIVE DEMO:
https://ademibowale.github.io/Capstone_projects/

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @ademibowale 

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ademibowale** [comment]
## In this Milestone

 - here is the loom video link  where i explain about this capstone project 
   https://www.loom.com/share/201a1c930c084c29a9f3cbf1198e59bb
 - capstone project for Entertainment industry
 - Implement JavaScript functionality dynamically
 - My code follows the style guidelines of this project
 - I have performed a self-review of my code
 - I have commented my code, particularly in hard-to-understand areas
 - I have made corresponding changes to the documentation
 - My changes generate no new warnings

**Pull request merged**